webxr-api/events.rs

```diff
@@ -0,0 +1,49 @@
+use crate::InputId;
+use crate::InputSource;
+
+#[derive(Copy, Clone, Debug)]
+#[cfg_attr(feature = "ipc", derive(serde::Serialize, serde::Deserialize))]
+pub enum Event {
+    /// Input source connected
+    AddInput(InputSource),
+    /// Input source disconnected
+    RemoveInput(InputId),
+    /// Session ended by device
+    SessionEnd,
```

Comment:
Should there be an event for the device being disconnected too? Blur / focus events?

Reply:
Same thing as SessionEnd from the POV of the device IMO.

Blur/Focus yes, I just haven't gotten to it yet.

Reply:
We should probably have VisibilityStateChange instead